ansible_base/activitystream/urls.py

```diff
@@ -1,12 +1,13 @@
 from django.urls import include, path
+from rest_framework import routers
 
 from ansible_base.activitystream import views
-from ansible_base.lib.routers import AssociationResourceRouter
 
-router = AssociationResourceRouter()
+router = routers.SimpleRouter()
```

Comment:
Out of curosity, why the change here? Is there something we need to/can do better in `AssociationResourceRouter`?